learning/katas/go/course-info.yaml

```diff
@@ -0,0 +1,26 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+title: Beam Katas - Go
+language: English
+summary: "This course provides a series of katas to get familiar with Apache Beam.\
+  \ \n\nApache Beam website – https://beam.apache.org/"
+programming_language: Go
```

Comment:
FYI there is an optional `programming_language_version` that I had tried to set with `1.13` and IntelliJ Edu kept throwing an error "programming language version not found"